dev-docs/bidders/axis.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: bidder
+title: Axis
+description: Prebid Axis Bidder Adapter
+biddercode: axis
+media_types: banner, video, native
+pbjs: true
+pbs: true
+safeframes_ok: true
+fpd_supported: false
+multiformat_supported: will-not-bid
+ortb_blocking_supported: partial
+usp_supported: true
+gdpr_supported: true
+coppa_supported: true
+schain_supported: true
+floors_supported: true
+sidebarType: 1
+---
+
+### Prebid Params
```

Comment:
If you have a working example or even some test parameters, this would be amazing to add here.

It's also okay to put a contact email in the docs for interested publishers.